### PR TITLE
fixes #34

### LINKED ISSUE #34: Option to add a date to the milestone
## [1.0.0] - 2017-06-20

### PR DESCRIPTION
**Pull request by prisis**
Added a new option `include-date` to the command
Added new functions to ChangelogConfig
Added a new option `date_format` to ChangelogConfig 

With the new option the new Changelog looks like this
```markdown
## v1.0.0 - 2018.1.1
```

### CODE REVIEW

File: src/ChangelogGenerator/Command/GenerateChangelogCommand.php [hunk 1]
```diff
@@ -351,4 +367,26 @@ private function getIncludeOpen(InputInterface $input) : bool
 
         return false;
     }
+
+    private function getIncludeDate(InputInterface $input) : bool
```
Comment (jwage) [inline src/ChangelogGenerator/Command/GenerateChangelogCommand.php]: Can we make a common private method and use it for both `getIncludeOpen` and `getIncludeDate`? They are identical except for the option name. Thoughts?

Comment (prisis) [reply src/ChangelogGenerator/Command/GenerateChangelogCommand.php]: If you like i can refactor this command or only merge both functions

Comment (jwage) [reply src/ChangelogGenerator/Command/GenerateChangelogCommand.php]: We should probably refactor so we're not duplicating code.

**Comment by jwage** [comment]
This needs to be rebased now.

**Comment by prisis** [comment]
Sorry, for the late reponse, need to find some time to finish this

**Comment by jwage** [comment]
@prisis If you are ok with it, I can finish it. If you get to it first, that is fine too. I might have time this upcoming week. Let me know. Thanks!

**Comment by WyriHaximus** [comment]
@prisis would you have time to rebase and update this PR? This is a great future to get into `v1.3`

**Comment by prisis** [comment]
Due to changes in interests, I will be closing this PR after several years.

**Pull request closed without merging**